data/schema/currencies.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
```

Comment:
Where did the list of currencies come from? We should add a description line to document what standard they follow e.g. ISO 4217